assets/javascripts/discourse/components/bcc-checkbox.js.es6

```diff
@@ -12,7 +12,7 @@ export default Component.extend({
       return (
         this.currentUser.staff &&
         this.creatingPrivateMessage &&
-        ((this.targetRecipients || "").split(",").length > 1 ||
+        ((this.targetRecipients || "").split(",").filter(Boolean).length > 1 ||
```

Comment:
A cheeky bugfix in a "coding standards" PR? 😜 (I'll allow it 😉)

Reply:
Yeh I can see this being a bug indeed :p But I do apply it as a coding standard rule xD

Although technically: `"".split(",").length // 1`  not > 1 so that's not a bug AFAICT.

Reply:
It does change the behavior (for the better) when given e.g. `","`, `",,"` etc. 😉